client/scss/components/_tags.scss

```diff
@@ -0,0 +1,15 @@
+.tags {
+  @include plain-list();
+  @include font('HNL5');
+  padding-bottom: $vertical-space-unit;
```

Comment:
Could we not turn this into a class called `vert-space` pr sp,thing and just add it to the HTML?